compiler/plugins/json_schema.ml

```diff
@@ -76,13 +76,14 @@ module To_json = struct
       (ctx : decl_ctx)
       (fmt : Format.formatter)
       (sname : StructName.t) =
+    let path, fields = StructName.Map.find sname ctx.ctx_structs in
     Format.pp_print_list
       ~pp_sep:(fun fmt () -> Format.fprintf fmt ",@\n")
       (fun fmt (field_name, field_type) ->
-        Format.fprintf fmt "@[<hov 2>\"%a\": {@\n%a@]@\n}"
+        Format.fprintf fmt "@[<hov 2>\"%a%a\": {@\n%a@]@\n}" Print.path path
```

Comment:
@EmileRolley this will impact you in the future. Now structs can be imported from external modules. @AltGr what is `path` here from structs from their own module? Does `Print.path` return `""`, or `"bla."` (with the `.`)?

Reply:
Yes the output here will be `ModuleName.var_name`

Reply:
Noted!